### DISCUSSION TITLE
Bug in fsic/user_subsys/axil_slav.v

### DISCUSSION THREAD

**Discussion by ruei7916** [bol-edu/HLS-SOC-Discussions#214] [General]
https://github.com/bol-edu/fsic_fpga/blob/main/vivado/vvd_srcs/caravel_soc/rtl/user/user_subsys/axil_slav/rtl/axil_slav.v

![marked 2Screenshot 2024-05-04 at 22-46-03 fsic_fpga_rtl_user_user_subsys_axil_slav_rtl_axil_slav v at main · bol-edu_fsic_fpga](https://github.com/bol-edu/HLS-SOC-Discussions/assets/56112123/9986b841-8d7f-4a80-b3d1-75153f1e6d8c)

不管user_prj_sel是多少，都會選到user project 0的axil bus，因此導致只有user project 0可以正常運作。
其中還有幾行程式碼完全重複
可能是複製貼上時忘記修改

**Comment by michael61112** [comment]
是的，這要改一下，不然READ Cycle會讀不到
